client/ChatterApp.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
```

Comment:
Refactor out a lot of the html (see pure rendering react components) and functions, and import them instead. This file is too big.